includes/checkout/template.php

```diff
@@ -248,7 +248,7 @@ function edd_get_cc_form() {
 				<span class="card-type"></span>
 			</label>
 			<span class="edd-description" id="card_number_desc"><?php esc_html_e( 'The (typically) 16 digits on the front of your credit card.', 'easy-digital-downloads' ); ?></span>
-			<input type="tel" pattern="[0-9]{13,16}" autocomplete="off" name="card_number" id="card_number" class="card-number edd-input required" placeholder="<?php esc_html_e( 'Card number', 'easy-digital-downloads' ); ?>" aria-describedby="card_number_desc" />
+			<input type="tel" pattern="^[0-9!@#$%^&* ]*$" autocomplete="off" name="card_number" id="card_number" class="card-number edd-input required" placeholder="<?php esc_html_e( 'Card number', 'easy-digital-downloads' ); ?>" aria-describedby="card_number_desc" />
```

Comment:
@cklosowski This is resolving a merge conflict because the pattern attribute changed upstream. I copied the new pattern, please make sure I'm not undoing that enhancement.